interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is like HTML and can do 95% of what html can use. its a way to combine html and javascript on the same working page in front of you. a syntax difference would be { } (curly braces) when calling on javascript functions. 
 
-Researched answer:
+Researched answer: JSX stands for JavaScript XML, it is a javScript syntax extension. while using JSX a expression is called with {expression}
 
 2. What is yarn? What file(s) are modified in a React application when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: I do not know for certain at this moment but I will research the answer and get back to you. But if I were to give it a guess I would say it gets my data ready to be used on react and creates the template. 
 
-Researched answer:
+Researched answer: yet another resource negotiator, it is a package manger and adds paackages from its tree of dependencies. 
 
 3. What is a React component?
 
-Your answer:
+Your answer: a react component is a easy way to get framework on a app, its like a skeloton app to be worked off of. 
 
-Researched answer:
+Researched answer: react component is the framework reusable components they are like lego blocks that you can put together. 
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: state is something as it is viewed currently at the moment before or after a change. values is what something is store as having and can be called on. props is stored informatin in a function that alows you to add properties to somehthing else. 
 
-Researched answer:
+Researched answer: state is mangened within the component, props get passed to the component. value attribute is used to get the value for a input field. 
 
 5. What is the DOM?
 
-Your answer:
+Your answer:I can not remember at this time. 
```

Comment:
It is a good thing to be transparent with your interviewer like this. I have used the "I not know at the moment but I will get back to you..." or the "I am not as familiar with this topic as I am with ..." lines in an interview and it left them with the impression that I will be accountable and I am able to communicate any barriers with solutions I may have.